app/controllers/invites_controller.rb

```diff
@@ -0,0 +1,32 @@
+class InvitesController < ApplicationController
+
+  before_action :require_user
+
+  def new
+    @invite = Invite.new
+  end
+
+  def create
+    @invite = Invite.new(invite_params)
+    @invite.user = current_user
```

Comment:
Instead of having 2 lines (10, 11) you could just have `@invite = Invite.new(invite_params.merge(user: current_user))`